### DISCUSSION TITLE
Travelling Salesman Problem on Indian cities

### DISCUSSION THREAD

**Discussion by navyagarwal** [networkx/networkx#6561] [Show and tell]
Hey,

I wanted to share a cool visualization I created using GeoPandas and NetworkX to solve the travelling salesman problem on Indian cities. The visualization shows the optimal path a salesman could take to visit all the cities in the dataset.

<img width="331" alt="image" src="https://user-images.githubusercontent.com/82928853/228065347-fc3e1295-2b35-4a27-a35a-84cbc927c93a.png">

I was wondering if something like this could be incorporated into the gallery examples under the geospatial section. (An example for the travelling salesman problem on a random graph already exists in the drawing section though) @dschult @MridulS

**Comment by MridulS** [comment]
This is indeed a nice viz @navyagarwal, but we do need to strike a balance for examples. As this is already covered in the current gallery examples I don't think we should be adding another one. But please feel free to maybe make a blog post or something, tweet about it. That always help with getting things out :) Feel free to link to this issue or our gallery examples when making the blog post.
